tests/testthat/test-add_risktable.R

```diff
@@ -1,100 +1,80 @@
-#' @title Specifications add_risktable.survfit
-#' @section Last updated by:
-#' Steven Haesendonckx
-#' @section Last update date:
-#' 20-OCT-2021
-
-# Specifications ----------------------------------------------------------
-
+#' @title Specifications test-add_risktable.R
+#' @section Last updated by: Tim Treis (tim.treis@@outlook.de)
+#' @section Last update date: 2021-11-02 18:37:49
+#'
+#' @section List of tested specifications
 #' T1. The function accepts a `ggsurvfit` object
 #' T1.1 No error when a `ggsurvfit` object is passed to the function
 #' T1.2 An error when a non-`ggsurvfit` object is passed to the function
 #' T2. The risktables are `ggplot` representations of the actual risktables
 #' T2.1 When no strata were specified, an artificial strata is displayed 'Overall'
 #' T2.2 The calculated values in the risktable are not affected by the transformation to a `ggplot`
-#' T2.3 The risktables are placed below the visR plot, in alignment with the x-axis of a visR plot without legend
-#' T2.4 The risktables are placed below the visR plot, in alignment with the x-axis of a visR plot with a legend
-#' T3. The output object is ggplot with additional class `ggsurvfit` and an attribute `components`
+#' The output object is ggplot with additional class `ggsurvfit` and attribute `components`
```

Comment:
Original tests were removed why?
Probably I wanted to write tests for these but I could not get vdiffr to work. Could you write tests for these?